### DISCUSSION TITLE
Hungarian sentences that should be supported / Támogatandó magyar mondatok

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#368] [Supported Sentences]
If you don't feel comfortable contributing to the YAML in this repository, you can post your **Hungarian** (`hu`) sentences that match [our supported intents](https://github.com/home-assistant/intents/blob/main/intents.yaml) below.

**Comment by nagyrobi** [comment]
:hungary:  Ide is jelezhetitek milyen mondatokat ismerjen fel a rendszer, ha nincs kedvetek YAML-ban szerkesztgetni és PR-okkal bajlódni.

**Reply by nagyrobi** [reply]
Innen: https://github.com/home-assistant/intents/discussions/329#discussioncomment-4601984

> Egyelőre az a cél, hogy pár alap-mondatot ismerjen fel a rendszer, lássuk, elvi szinten jó irányban haladunk-e. Utána jöhet a bővítés. Most még nem érdemes túlbuzogni mindenféle mondat kombinációval mert esélyes, hogy mindent újra kell majd írni...

**Reply by cociweb** [reply]
Hi,
Iff the PoC is ready, I would like to recommend some audio related sentences into the backlog. Optionally with the device location (with area addition) and from a specific source (eg from tts announcement or from Music Assistant > tune-in channel, YTMusic playlist, etc.)
Eg.:
Némítsd el a konyhában a zenét|bejelentést|vinnyogást(tone bejátszás esetén)
Halkítsd le / hangosítsd fel /állítsd be a hangerőt 20%-ra a konyhai zümmögőn.
Indítsd el az  XY Youtube lejátszási listát/ XY Tunein csatornán.
Következő (elem a queue-ban) az összes/Group1 lejátszón.

 - First let's see, how will it works: Maybe it will remain a dream, because these are not strictly ha's parameters.

**Reply by cociweb** [reply]
Dear @nagyrobi,
I've just realized that **'kapcsold fel/be/le/ki a(z) {entity}-t/át/ét'** or the **'kapcsold fel/be/le/ki a(z) {entity} világítást/fényt/villanyt'** or the **'kapcsold fel/be/le/ki a(z) {entity} világítást/fényt/villanyt a(z) {area}-ban/ben'**  type sentences are not recognized by the assist. Can I kindly request to add them into the lightings?

**Reply by nagyrobi** [reply]
Your PRs are welcome.

**Reply by cociweb** [reply]
Well, I thought, this discussion thread is for an idea collector place where "ha a nincs kedvetek YAML-ban szerkesztgetni és PR-okkal bajlódni". - But it seems, I was wrong. ;)

**Reply by nagyrobi** [reply]
Not wrong. This is a community effort to maintain the software. At the moment it looks like nobody has time to handle Hungarian, but anyone can contribute for a common goal.

**Comment by vadaszi** [comment]
Hi!

Should we use exactly the same phrases than in English or could we simplify them? If yes, I suggest use some basic phrase which not use any "hard Hungarian grammar".
For example: Instead of "Kapcsold fel a villanyt a nappaliban" use only "nappali villany fel". 
I don't know how the language recognition works, but it's a simpler form. Clearly these are not the best Hungarian sentences but do the job.

**Reply by nagyrobi** [reply]
The system will likely support both approaches. Currently the goal is to see how the principle works, and the texts filled in pass the tests. If it passes that "hard Hungarian grammar", adding simplified versions will be a piece of cake.

**Comment by v1k70rk4** [comment]
-==EN==-
I've finally managed to finalize the Pull, I hope it passes the big test too :)
On paper, it seems to recognize a lot of Hungarian sentences, I'll be curious to see how it performs live :)

I hope it works well :)
And I welcome any ideas or suggestions for improvement.

-==HU==-
Na végre sikerült véglegesíteni a Pull-t, remélem a nagy teszten is átfut :)
Papíron elég durván sok magyar mondatot fel fog ismerni kíváncsi leszek élőben :)

      - kapcsold be a világítást a nappaliban
      - kapcsold föl az összes lámpát a nappaliban
      - Az összes lámpát kapcsold be a nappaliban
      - A nappaliban minden lámpát kapcsolj fel
      - A nappaliban az összes lámpát kapcsold fel
      - "Kapcsold be a Party mód jelenetet"
      - "Aktiváld a Party módot"

ami elragozható el van ragozva, ami nem ott kerülő szavakat tettem, hogy ne kelljen ragozni.
pl nappali szoba -> nappali szobaban lenne, de ha használható a nappali szoba helyiség/szobában 
```
  name: "[a |az ]{name}[<name_ragok>| <name_szavak>]"
  name_ragok: "(n(a|e)k|[o]t|r(a|e))"
  name_szavak: "((eszköz|érzékelő)[nek|t|re]|szenzor[nak|t|ra])"
  area: "[a |az ]{area}[<area_ragok>| <area_szavak>]"
  area_ragok: "(b(a|e)[n]|(o|e|ö)n|t|i)"
  area_szavak: "([ ](szob(a|á)[ba[n]|t|i]|hely[i]ség[be[n]|en|et]))"
```


érdemes lesz a szoba szót kivenni majd és ha csak a nappali van akkor lehet hivatkozni rá
nappaliban -> nappali szobában -> nappali helyiségben néven.

én pl a 'konyha' szónak felvettem a 'konyhá' aliast így lehet mondani hogy konyha -> konyhában etc....

remélem jól fog működni :)
és ha van ötletek javítások várom szeretettel.

**Reply by nagyrobi** [reply]
Komoly munka, köszönjük!!!

Majd mesélj milyen hardvert használsz hozzá...

**Reply by v1k70rk4** [reply]
A gép egy leselejtezett vékony kliens :) x64-es i5 7gen 8Gb ram
Oprendszer: HAOS

Nekem a lokális STT (whisper) a fenti konfigon egy katasztrófa :( csak a small vagy medium-int8 működik és egyik rosszabb mint a másik... Ezért a felhős hangfelismerésnél maradtam :) 

Ami elírások vannak a kész felismerésben az a Microsoft STT elírásai, így mivel az áll rendelkezésre arra van optimalizálva. ezért van egy csomó összetett szóban egy fakultatív szóköz [ ] jellel és a hely[i]ség is azért szerepel így, mert akármennyire szépen mondtam ki mindig helység lett (na meg sokan nem is tudják, hogy az más)  :)

Pluginek: 
- Repó: https://github.com/hugobloem/homeassistant-addons
  - Hangfelismerésre:    https://github.com/hugobloem/wyoming-microsoft-stt
  - Szövegfelolvasásra: https://github.com/hugobloem/wyoming-microsoft-tts (tamás és noémi neurális hangokkal)
- Repó: https://github.com/rhasspy/hassio-addons
  - USB mikrofon használatára: https://github.com/rhasspy/hassio-addons/tree/master/assist_microphone

Beépített:
  - OpenWakeWord: https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_openwakeword (még most kezdek neki)

Az STT és TTS-ből is elérhető ingyenes verzió az STT-ből 5 óra havonta a TTS-ből meg az apámék rendszere még nem futott ki, pedig mindent IS felolvas nekik a rendszer :) De a fizetős se eszement árú.  nekem 1-2$ szokott lenni a számlám és nem keveset használom :)  (NorthEurope zónát ajánlom!)
https://azure.microsoft.com/hu-hu/pricing/details/cognitive-services/speech-services/
https://azure.microsoft.com/hu-hu/pricing/details/cognitive-services/speech-services/

**Reply by nagyrobi** [reply]
Nálam a HA a pincében van, egy VM-ben... szó se róla hogy ahhoz a vashoz beszéljek... [ESPHome-alapú Voice Assistant](https://esphome.io/components/voice_assistant.html) megoldást nem próbáltad? [Pl. így](https://gist.github.com/tetele/75815c53cfd2a02412ef153f9c2d24e2).

Majd valaki csinálhatna egy step-by-step leírást (nem videót!) magyaroknak.

**Reply by v1k70rk4** [reply]
Nem próbáltam, de kolléga - ha működni fog az assist éles rendszerben is - szerintem neki ugrik az esp-home-os megoldásnak mert neki több szobában van rá szüksége. Én csak a nappaliban kívánok vele beszélgetni.

Gondolkodtam én is a VM-en meg a szerveren, de ahhoz nem tudnék akkora szünetmentest tenni :) 

Vékony kliensből van kettő, ha bedöglik a másikat csak átteszem és rátöltöm a mentést. A teljes hálózatos sarok eszik kb 40W-ot így 5 órát elüzemel a szünetmentesről és így a szünetmentesen lévő világítás is megy rendesen áramszünet esetén (max ilyenkor lekapcsol 10 percenként a lámpa :) de van).  Muszáj a redundancia, mert nincsenek fizikai kapcsolóim a fűtés/hűtés/légtisztítás/párásítás/párátlanítás is erről megy.

Hát videót én tuti nem fogok csinálni, de configokat osztok meg szívesen, az ESP homeos konfiggal meg ha előre jutunk kollégával akkor ide beírom.

**Reply by nagyrobi** [reply]
Ha kollégának van már elképzelése milyen hardvert akar ESPHome-os változatban dobjatok már egy sort akár Discord-on nekem, rendelnék én is tesztre vmit.

**Reply by solazs** [reply]
Nagyon koszi a munkatokat! :) Lehet tudni, hogy ez a mostani PR mikor eri el az eles HA-t?

Kiserletezek a lokalis STT-vel, most ott jarok, hogy lecsereltem a default modellt a rhasspy/wyoming-whisper-ben masikra (https://huggingface.co/Hungarians modellein futtatva a ct2 converter scriptet innen: https://github.com/SYSTRAN/faster-whisper), de sajnos nekem is az a tapasztalatom, hogy nem tul pontos. A kapcsold szot peldaul konzisztensen kapcsolt-nak hallja, nappali helyett napali stb... Sajnos videokartyat nem tudok epp szolgalatba allitani, de talan az lesz a kovetkezo.
Vagy en is adok egy probat az azure-os megoldasnak, az egyetlen GPUm passthrough-zva van, nem szivesen bontom szet a konfigot.

A whispert egyebkent halozaton keresztul futtatom a nagyobbik geprol (i5 7900, sok ram). A pontossag a nagyobb modellekkel sem javul, cserebe a sebesseg ertelemszeruen rosszabb. Okosoraba/telefonba beszelek, igy nem szamit hol vannak a gepek.

TTS-re jol mukodik a piper lokalisan, egy negyedik generacios i5-on fut a HA nalam (regi laptop -> beepitett UPS! :), szinte alig latszik a terhelesen.

**Reply by v1k70rk4** [reply]
Ahogy nézem hónap elején készülnek a csomagok. Már én is várom nagyon :) 
Jelenleg dev home assistanttal tesztelgetem egész jónak néz ki.

Nekem a piper hangja se tetszik :( De lehet én vagyok túl körülményes :)

**Reply by solazs** [reply]
Milyen hangot próbáltál?

Én ezt állítottam be: https://huggingface.co/rhasspy/piper-voices/tree/v1.0.0/hu/hu_HU/imre/medium

Egész érthetően beszél. Néha a hangsúlyozás fura, de mégis csak egy gép :)

Én megvárom a hó elejét, de kíváncsi vagyok hogy fog működni.
Párom nagyot nézett, mikor visszaszólt az óra, még az is lehet, hogy megtetszik neki a dolog :)

**Reply by v1k70rk4** [reply]
Rég próbáltam, lehet hogy fejlődött azóta, mert határozottan rosszabbra emlékszem.
Amúgy lehet hogy csak Annát néztem annó, mert ő most is kihagyott betűket és rossz az intonáció.
Kicsit Imre meg tompa rádiós hangú és valóban rosszul intonál és mintha meg lenne fázva :D
Berta viszont tetszik ha őt hallgatom először akkor talán nem lesz Microsoft TTS :)
Köszi az oldalt.

Az én párom nagyon imádja :D nem engedi visszatenni a stabil release-t :D én meg fázok a Dev-től... csak a dev VPS-en meg nem tudok zigbee cuccokat tesztelni és kb minden is zigbee-s.

**Reply by cociweb** [reply]
A Berta valóban életképesebb a többieknél. :)

@solazs, a hugginface-en a modelleket elő kvantáltuk az almappákba. (ugyan azt a konvertert használva, amit linkeltél. :) ) - Érdemes egyébként kipróbálni a különböző verziójú, egyazon modelleket: pl nekem a tiny-v2 jobban bejött mint a v3, vagy a final (hiába jobbak a wer/loss értékek a tanulásnál) és azon belül is értelem szerűen az fp32 (konvertálás nélküli) modellek jobb eredményt adnak.

Ugyanakkor tessék figyelembe venni, hogy a whisper egy profitorientált cég, ingyenesen elérhetővé tett 'demo'-ja, ami mostmár lassan közelebb van a 2 éves életkorhoz, mint a 1.5-hez. Nyílván, még sok sebből vérzik, vannak gyerekbetegségei (pl a vocabulary nem tartalmaz magyar szavakat, stb). Ez afféle kereskedelmi méregetés, "hogy mi már itt járunk. és Ti?" - csakhát egyelőre más még nem adott ki ingyenesen jobbat....

**Reply by solazs** [reply]
@cociweb

Először is semmiképp sem szerettem volna becsmérelni bárki munkáját, meglepett és őszinte örömmel töltött el, hogy mennyi jó (és nem utolsó sorban magyar) opció áll már a magamfajta 'gyüttment' ember rendelkezésére.

Hálás vagyok mindenkinek, aki munkát öl a témába és ingyen elérhetővé teszi az eredményét :)

A dologban az is benne van, hogy nem értek az egészhez, ráadásul a nagy modellekhez kevés a hardverem. Közben rájöttem arra is, hogy feleslegesen járattam itthon a konvertert, csak azzal nem jöttem még ide dicsekedni :) 
Köszönöm, hogy szóltál!

Egyébként Fp32-vel a large modell esetén _már majdnem jó_, csak kevés a hardverem a real-time futáshoz.

Majd próbálkozok még, csak nem volt időm ezzel foglalkozni.

**Comment by bithajcsar** [comment]
Sziasztok!
Először is: köszönet mindenkinek, aki összerakta(ja) nekünk a magyar nyelvű mondatokat! Én a minap kezdtem bele, hogy a Home Assistant-ommal beszélgetni tudjak. Bevallom, némely apró sikertől eltekintve, lehangoló az élmény, pedig a Home Assitstant Cloud-ra bíztam a beszéd felismerést és szintézist. Ha ügyesen (STT számára könnyen felismerhető szavakkal) adom meg az objektumaim neveit és álneveit (alias), akkor azokat felismeri a rendszer (M5Stack Atom Echo). De a "bekapcsol" és a "kikapcsol" utasításokon kívül semmit sem ért meg. A bonyolultabb mondatok teljesen esélytelenek. Úgy tűnik, hogy valamit rosszul csinálok. De mit? Ha valakinek közöttetek van ötlete erről azt nagyon szívesen venném.
Bár nem igazán illik ide, de gyakorlott programozók jelenlétét feltételezem ezen a fórumon, úgyhogy, hátha... Megkérdezem:
Mivel a hívószó felismerése igen csak esetleges, és az eredményt csak az Atom Echo fénye (villogó kék) jelzi, ezért próbálkoztam azzal, hogy amint érzékelte a hívószót, mondjon valamit (Mit tehetek önért?). Eddig nem jártam sikerrel. Szívesen vennék valami iránymutatást erről is!
Előre is kösz!
Bithajcsár

**Reply by nagyrobi** [reply]
Várd meg a következő release-t...

**Reply by bithajcsar** [reply]
Na... Most izgulhatok! :rofl:
Kösz!

**Comment by v1k70rk4** [comment]
Átírtam a válaszokat :) a névelőket már kezeli így elég értelmes magyar mondatok jönnek majd ki, de a ragozásra ez jutott eszembe :D
```
{% macro ragozas(szoveg) %}
  {% set mely_maganhangzok = "aáoóuú" %}
  {% set magas_maganhangzok = "eéiíöőüű" %}
  {% set utolso_maganhangzo = "" %}
  
  {% for char in szoveg %}
    {% if char in mely_maganhangzok or char in magas_maganhangzok %}
      {% set utolso_maganhangzo = char %}
    {% endif %}
  {% endfor %}

  {% if utolso_maganhangzo in mely_maganhangzok %}
    {{ szoveg }}ban
  {% elif utolso_maganhangzo in magas_maganhangzok %}
    {{ szoveg }}ben
  {% else %}
    {{ szoveg }}
  {% endif %}
{% endmacro %}

{{ ragozas("kert") }}  {# kimenet: kertben#}
{{ ragozas("cukrászda") }}  {# kimenet: cukrászdaban #}
```
de marad a 'Józsi az Iroda zónában van' 
mert azt már végképpen nem tudom lekezelni, ha változik a vége a szónak.

de jól el szórakoztam vele :D

**Reply by Andreaux** [reply]
Ez marha izgi... viszon cukrászdaban heylett cukrászd**á**ban lenne, nem?

**Reply by v1k70rk4** [reply]
Igen pont ezt írtam, hogy azt nem tudom kezelni  sajnos :( Jina2-ben nem tudok törölni, meg szabályt se tudok rá program nyelven, hogy mikor változik a szótő. Ehhez már valami nyelvi modell kéne vagy szótár.  De egyik sem megoldható ezen keretek között.

**Reply by Andreaux** [reply]
Ajajj... pedig pont ez a sava borsa :D Kutyafáját...
Nem lehet egy nyelvi modellt közbeiktatni, hogy értelmezze a mondatot és alakítsa olyanra, amit a HA csont nélkül megesz? Nagyon elvetemült ötlet?

**Reply by Andreaux** [reply]
például a " |truncate(1, True) " sem használható erre? Bocs, ha buta ötlet.

**Reply by v1k70rk4** [reply]
de miből mondod meg, hogy egy szónak változik a szótöve? :) Amúgy annyira nem nagy probléma ez, mert töltelék szavakkal ki lehet játszani. Kevés válasz van és az is nagyon nagyon egyszerű.

Majd használat közben látni fogod, hogy a hibaüzenetek és a magyar nyelv nem barátok, de az lenne a lényeg, hogy kevés hibaüzenet legyen :) Ezer megoldást próbáltam a szavak szótövére találjak megoldást, de nincs a nyelvi korlátok és a számítási korlátok miatt. (Futnia kell mindennek a Pi2-őn)

**Reply by Andreaux** [reply]
Hát, a feleségem neurolingvisztikát tanult nyelvész... ha beállítom neki Bertát karattyolni, két perc alatt megfejti nekünk hogy lehet, annyira fogja idegesíteni :D:D

**Reply by v1k70rk4** [reply]
A hibaüzenetekben a kérdés ragozásától függ a nyelvi hiba, de a válaszokban nincs nyelvi hiba! Így remélhetőleg nem lesz gondja :)

**Comment by v1k70rk4** [comment]
Eljött a nagy nap :) a 2024.2.0b4 -ben már benne vannak a magyar dolgok. (ezt már élő rendszerre is felmerem tenni :) )
Frissítéseknél jobb felső sarok és csatlakozz a bétához. Elég csak a core-t updatelni.
A végleges megjelenés Február 7-én lesz.

Teszteljétek és írjátok a hibákat, nekem néha elég fura dolgok jöttek ki, de amit tudtam javítani a hamis egyezések elkerülésében azt beleírtam.

Fontos még, hogy a kényelmes használathoz azokat a helyiségneveket amiknek változik a szótöve pl.: konyha (konyhában) iroda (irodában) érdemes felvenni aliasnak a konyhá és az irodá változókat is. Bár használható például a konyhai ragozott alak és a konyha helyiségben alak is, de nekem a konyhában áll a számra :)
Ha úgy van elmentve hogy Nappali szoba érdemes csak a nappali aliast feltenni és használhatod utána szoba minden ragozott alakját.

```
expansion_rules:
  name: "[a |az ]{name}[<name_ragok>| <name_szavak>]"
  name_ragok: "(n(a|e)k|[o]t|r(a|e))"
  name_szavak: "((eszköz|érzékelő)[nek|t|re]|szenzor[nak|t|ra])"
  area: "[a |az ]{area}[<area_ragok>|<area_szavak>]"
  area_ragok: "(b(a|e)[n]|(o|e|ö)n|t|i)"
  area_szavak: "([ ](szob(a|á)[ba[n]|t|i]|hely[i]ség[be[n]|en|et]))"
```

A hibabejelentés módja akár itt, akár hibajegyen:
- Írd le a kérdést
- a kapott választ
- és a fejlesztői eszközökben az assist fül alatt kapott elemzést CODE formátumban:
```
intent:
  name: HassTurnOn
slots:
  name: ikeás lámpán a színes fény
  domain: scene
details:
  name:
    name: name
    value: scene.papir_lampa_tv
    text: ikeás lámpán a színes fény
  domain:
    name: domain
    value: scene
    text: ''
targets:
  scene.papir_lampa_tv:
    matched: true
match: true
sentence_template: ((<set>|<turn_on>);<name> [a] jelenetet)
unmatched_slots: {}
source: builtin
```

**Comment by CiniBN** [comment]
Sziasztok!

Nagyon köszönjük az eddig elvégzett munkátokat!

Lenne egy kérésem:
"Állítsd a lámpa színhőmérsékletét xxxx-ra(re)"
mondatot kérem felvenni. Sajnos az angol mondatok között nem találtam kifejezetten a színhőmérsékletre utaló szándékot, lehet, hogy nem is létezik?

Köszönöm!

**Reply by v1k70rk4** [reply]
Szia, jelenleg az összes támogatott mondat benne van. 
Sőt több is, mert még a HassClimateSetTemperature és a person_HassGetState ág sincs engedélyezve jelenleg :(

**Reply by CiniBN** [reply]
Tehát, ha az angolban nincs színhőmérsékletre utaló szándék, akkor a magyarban egyenlőre nem fog szerepelni?

**Reply by v1k70rk4** [reply]
így van sajnos. Jelenleg nem támogatja az alap motor a színhőmérséklet állítást. Színekből se tudok többet beleírni mint ami az angolban benne van (kipróbáltam :) ). 

Maximum jelenettel lehet próbálkozni, hogy elmented jelenetnek és azt aktiválod én így használom a lámpák szín/színhőmérséklet állítását.

**Comment by Andreaux** [comment]
Sziasztok!
Nemrég kezdtem barátkozni ezzel az egésszel és fel is telepítettem a whisper/piper/openwakeword kombót. Egész ígéretesek a dolgok ha írásban kommunikálok a segéddel, ellentétben azzal, hogy mobilról totál semmit sem ért abból, amit mondok (mintha angolul próbálná értelmezni a magyar szöveget, pedig be van állítva neki, hogy magyar). Gépen próbálva úgy ahogy néha elkapja, amit akarok, de akkor sem ért semimt belőle. "Kapcsold le a lámpát a konyhában" mondatra "Sajnálom, a konyában. terület nem található"-val válaszol (direkt felvettem "konyha" és "konyhá" aliasokat a konyhára). Mit csinálok rosszul?

**Reply by v1k70rk4** [reply]
Szia, örülök, hogy idetaláltál :) 
A whisper én is küzödök, nekem sehogy se jó :( Én a Microsoft STT-t használom erre a célra.
A lámpa problémád Ez a hiba nem egész 10 perc múlva javítva lesz ;) https://github.com/home-assistant/core/actions/runs/8176862483 amint ez lefut frissítheted majd a 2024.03-ra és abban, ez helyesen fog működni.

**Reply by Andreaux** [reply]
Én is örülök, hogy rátok találtam. Olyan izgi ez az egész :D Lenyűgöző mennyi melót beletettetek már ebbe a magyarításba. Szeretnék valahogy segíteni tudni, legalább ötletekkel :)

A Microsoft STT helyben fut és megy Linux-on is? Nem ismerem. Azért futottam a Whisper-nek neki, mert mindent helyben szeretnék tartani és Linux VM-ben futtatni.

**Reply by v1k70rk4** [reply]
Hát a Microsoft STT sajnos felhős :(  a Whisperből van olyan ami GPU-n fut és az egész jól felismeri a magyar szöveget a high modellel, de nem voltam hajlandó erre egy videókártyát beáldozni :)
Ehhez talán a minimum egy GTX 2060 kell. Sem fogyasztásban nem tetszik, se a vékonykliensemiet sem akarom lecserélni :P

**Reply by Andreaux** [reply]
Nálam csak tervben van, de még nincs GPU a szerverben, viszont magok meg ram van bőven... meg valahogy nem az az érzésem, hogy kevés, hanem, hogy valami alapvetően nincs jól beállítva... főleg mobil esetén, tényleg olyan, mint amikor a Siri-nek magyarul akarok diktálni, de a billentyűzet angolon maradt...

**Reply by v1k70rk4** [reply]
Segítség: sajnos a rendszerbe nem az kerül bele amit kitalálunk, így ez leginkább fordítás és feltételez egy közepes jina2 ismeretet is, de ha valami eszedbe jut írhatsz is ide vagy nyithatsz egy PR-t. De érdemes átnézni a kódot, hogy mik a limitek.

A homeassistant fordításhoz itt tudsz csatlakozni: https://developers.home-assistant.io/docs/translations
ide nem kell kódolási ismeret, de itt is érdemes figyelni, hogy egységes legyen a nyelvezet.

**Reply by v1k70rk4** [reply]
Tudomásom szerint CPU-val csak a medium készlet elérhető és az szerintem nem alkalmas magyar szöveg hibamentes felismerésére, de nézd meg a whispered mire van beállítva, lehet csak a small van engedve. Mert annyira viszont nem rossz, mint ahogy írod.

**Reply by Andreaux** [reply]
"--model tiny-int8 --language hu" :D Gondolom akkor itt van a kutya elásva :D Na mindjárt meg is nézem hogy kell nagyobbat.

**Reply by v1k70rk4** [reply]
így így ez a hajnali harmatos teveszar kategória. Szerintem neked a medium lesz a max és ha van sok szálad akkor adj neki 4-et :)

Közben kijött a 2024.03 is :)

**Reply by Andreaux** [reply]
48 van, csak jut... na proxmox backupolja a vm-et és már ugrok is át a béta ágra (eddig nem azon voltam)... ez béta ág, ugye?

**Reply by Andreaux** [reply]
Óóó, látom nem béta.... de jó :) Indítom is az update-et... Köszönöm!

**Reply by v1k70rk4** [reply]
a Beta ág az mindig hónap vége előtt indul általában 7-8 kiadás jön a főverzió cseréknél. Ez a ciklus most zárult így most a beta ágon ugyan az van.

**Reply by Andreaux** [reply]
Durván tökéletes volt akkor az időzítésem... lenyűgöző :)
